### ISSUE TITLE
Batch Norm issues

### ISSUE THREAD

**Issue by MOCR** [hans/rlcomp#2]
Hello there!

I am trying to implement DDPG as well, but I am having a hard time trying to add batch norm to the actor and critic networks, i get really bad results with bn (diverging networks, or having trouble converging...). Did you managed to get a working version with bn (results at least as good as without?).

I am pretty new to neural networks, so I may have missed something... 

(I am using tensorflow as-well)

Thanks.

**Comment by hans** [comment]
Hi @MOCR,
I did not use batch-normalization in my implementation of the standard DDPG. I was able to get okay results without BN on the small cartpole-swingup task. I tried batch norm for my extension of DDPG to sequences, but it had very little effect on training stability in preliminary experiments. I didn't run ever run a large suite of complete experiments with BN enabled, but I expect they would not have done much better than the non-BN experiments.

Hope that helps.
